activation/activation.go

```diff
@@ -135,133 +139,69 @@ func (b *Builder) Start() {
 
 // Stop stops the atx builder.
 func (b *Builder) Stop() {
```

Comment:
stop should unsubscribe from waiting on atx channel

Reply:
👍 done